lib/ansible/modules/crypto/luks_device.py

```diff
@@ -72,15 +82,26 @@
     new_keyfile:
         description:
             - "Adds additional key to given container on I(device).
-              Needs I(keyfile) option for authorization. LUKS container
-              supports up to 8 keys. Parameter value is the path
-              to the keyfile with the passphrase."
+              Needs I(keyfile) or I(passphrase) option for authorization.
+              LUKS container supports up to 8 keyslots. Parameter value
+              is the path to the keyfile with the passphrase."
             - "NOTE that adding additional keys is I(not idempotent).
               A new keyslot will be used even if another keyslot already
               exists for this keyfile."
             - "BEWARE that working with keyfiles in plaintext is dangerous.
               Make sure that they are protected."
         type: path
+    new_passphrase:
+        description:
+            - "Adds additional passphrase to given container on I(device).
+              Needs I(keyfile) or I(passphrase) option for authorization. LUKS
+              container supports up to 8 keyslots. Parameter value is a string
+              with the new passphrase."
+            - "NOTE that adding additional passphrase is I(not idempotent).  A
```

Comment:
```suggestion
            - "NOTE that adding additional passphrase is *not idempotent*.  A
```
Can you also change the same thing in the previous option?